addons/zeus/CfgVehicles.hpp

```diff
@@ -317,4 +317,35 @@ class CfgVehicles {
     class ModuleArsenal_F: Module_F {
         function=QFUNC(bi_moduleArsenal);
     };
+
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class GVAR(create) {
+                displayName = CSTRING(CreateZeus);
+                condition = QUOTE(\
```

Comment:
```
14:09:37 File z\ace\addons\zeus\CfgVehicles.hpp, line 325: '/CfgVehicles/CAManBase/ACE_SelfActions/ace_zeus_create.condition': Missing ';' at the end of line
14:09:37 ErrorMessage: File z\ace\addons\zeus\CfgVehicles.hpp, line 325: /CfgVehicles/CAManBase/ACE_SelfActions/ace_zeus_create/: Wrong # directive
14:09:37 Application terminated intentionally
ErrorMessage: File z\ace\addons\zeus\CfgVehicles.hpp, line 325: /CfgVehicles/CAManBase/ACE_SelfActions/ace_zeus_create/: Wrong # directive
```
I think complex code like this should just be in a function, I hate dealing with ARR_X and string escaping

Reply:
Is that mikeros tools? Might be a bug in his parser ^^
Yeah I'll move it.. Name?
zeus_fnc_actionCreateCondition ?

Reply:
It's a condition, so `zeus_fnc_canCreate` (something with `canX`).

Reply:
it was crashing arma on game start
the way it's now handled in master is good enough imho (https://github.com/acemod/ACE3/blob/master/addons/zeus/XEH_postInit.sqf#L85-L121)
otherwise `zeus_fnc_actionCreateCondition`/`zeus_fnc_actionCreateStatement` isn't bad because they are sorted alphabetically

Reply:
The code in current master will still execute code when you spawn units in Zeus right? Even if just once per type. Whereas the config one wouldn't even do that?
Also it seems like with the script code we'll be adding it to tons of CAManBase based classes, even though we really only need CAManBase once.

I checked the config code using the make script, Don't remember whether mikero or armake.

Reply:
no code happens on object init
only fires once each time the player controls a new man/vehicle type